client/src/main/java/com/tigrisdata/db/client/search/HitMeta.java

```diff
@@ -14,19 +14,94 @@
 
 package com.tigrisdata.db.client.search;
 
+import com.google.protobuf.Timestamp;
 import com.tigrisdata.db.api.v1.grpc.Api;
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
 import java.util.Objects;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
 
 /**
  * Representation of {@link Api.SearchHitMeta} exposing relevance information for the matched
  * document.
  */
 public final class HitMeta {
-  // TODO: Implement once API structure is finalized
-  private static final HitMeta DEFAULT_INSTANCE = new HitMeta();
+  private final OffsetDateTime createdAt;
+  private final OffsetDateTime updatedAt;
 
+  private HitMeta(OffsetDateTime createdAt, OffsetDateTime updatedAt) {
+    this.createdAt = createdAt;
+    this.updatedAt = updatedAt;
+  }
+
+  /**
+   * Gets the time at which document was inserted/replaced to a precision of milliseconds in {@code
+   * ZoneOffset.UTC}
+   *
+   * @return time in UTC or null
+   */
+  public OffsetDateTime getCreatedAt() {
```

Comment:
DMLResponse exposes similar information as `protobuf.Timestamp`, should we instead present that as a Java native data type - `OffsetDateTime` or `Instant`?

Reply:
`Instant` sounds good 👍